deployer/src/deployment/queue.rs

```diff
@@ -79,6 +88,17 @@ pub async fn task(
                             Err(err) => return build_failed(&id, err),
                         }
 
+                        match builder_client.build(BuildRequest {
+                            deployment_id: queued.id.to_string(),
+                            archive: queued.data.clone(),
+                        }).await {
+                            Ok(inner) =>  {
+                                let response = inner.into_inner();
+                                info!(deployment_id = %queued.id, "shuttle-builder finished building the deployment: image length is {} bytes, is_wasm flag is {} and there are {} secrets", response.image.len(), response.is_wasm, response.secrets.len());
+                            },
+                            Err(err) => error!(deployment_id = %queued.id, "shuttle-builder errored while building: {}", err)
```

Comment:
Will these show up in Deployer logs to user? They don't need to yet.

Reply:
Hmm, like it is currently I think they will. Perhaps if we instrument the span when we spawn this and don't include the deployment_id we will skip sending it in the deploy layer? Or rename the deployment_id.

Reply:
I renamed the `deployment_id` for now. We can look further into it once it's on staging. In parallel I'll continue my local testing.

Reply:
Ah, okay, I think this will work, but we'll need to see in testing.